src/components/10-atoms/toggle-switch/index.js

```diff
@@ -73,13 +80,10 @@ class AXAToggleSwitch extends LitElement {
         ${inputElement}
         <div class="a-toggle-switch__slider"></div>
       </label>
+      <span class="${classMap(classes)}">${error}</span>
```

Comment:
Please use axa-text instead of span to have correct font-style. And please update `input-text` and all other components which displays an error text (or create a issue for that). I saw that i[nput-text also uses span](https://patterns.axa.ch/?path=/story/components-input-text--input-text&knob-label*=&knob-name*=&knob-refid=&knob-placeholder=&knob-value=&knob-currency=&knob-error=asdf&knob-info=&knob-checkmark=&knob-disabled=&knob-required=&knob-invalid=true&knob-type=text&knob-maxlength=50&knob-counter=Stillcharacterseft&knob-counterMax=Overcharacterlimit!&knob-pattern=&knob-inputmode=&knob-autofocus=), which is not looking good.

Reply:
The problem with using `<axa-text>` is, that you have to modify it (ex: `font-size`, `letter-spacing`, etc) because the stock `<axa-text>` has different properties than what we need for this case.

Reply:
have you discussed this with the designers?